src/recordlinker/routes/patient_router.py

```diff
@@ -43,24 +48,29 @@ def create_person(
     "/{patient_reference_id}/person",
     summary="Assign Patient to existing Person",
     status_code=fastapi.status.HTTP_200_OK,
+    deprecated=True,
 )
 def update_person(
     patient_reference_id: uuid.UUID,
     data: schemas.PersonRef,
     session: orm.Session = fastapi.Depends(get_session),
 ) -> schemas.PatientPersonRef:
     """
+    **NOTE**: This endpoint is deprecated. Use the PATCH `/person/{person_reference_id}` endpoint instead.
+
+    **NOTE**: This endpoint will be removed in v25.2.0.
+
     Update the Person linked on the Patient.
     """
-    patient = service.get_patient_by_reference_id(session, patient_reference_id)
+    patient = service.get_patients_by_reference_ids(session, patient_reference_id)[0]
```

Comment:
If we're just pulling one patient out of the returned bunch, that would seem to support having a function that does a single patient operation as the correct signature, right?

Reply:
Yes, however this method is deprecated and will be removed in a future version.  Which reminds me I need to bump that value to v25.3.0.  I didn't bother to leave the old get_patients_by_reference_id function, because we're just going to remove this route in a couple of weeks.